packages/nimble-components/package.json

```
@@ -87,6 +91,7 @@
     "@types/d3-selection": "^3.0.0",
     "@types/d3-zoom": "^3.0.0",
     "@types/markdown-it": "^13.0.0",
+    "comlink": "^4.4.1",
```

Let's pin the version to prevent the chance that the main thread and the pre-built library get out of sync.

And don't forget to run npm install to get the package-lock update after

```suggestion
    "comlink": "4.4.1",
```